code/modules/mob/living/simple_mob/subtypes/vore/rabbit.dm

```diff
@@ -0,0 +1,150 @@
+/datum/category_item/catalogue/fauna/rabbit		//TODO: VIRGO_LORE_WRITING_WIP
+	name = "Wildlife - Rabbit"
+	desc = ""
+	value = CATALOGUER_REWARD_TRIVIAL
+
+/mob/living/simple_mob/vore/rabbit
+	name = "rabbit"
+	desc = "It's a fluffy, smol bunny! Adorable!"
+	tt_desc = "Vulpes zerda"
+
+	icon_state = "rabbit_brown"
+	icon_living = "rabbit_brown"
+	icon_dead = "rabbit_brown_dead"
+	icon_rest = "rabbit_brown_rest"
+	icon = 'icons/mob/vore.dmi'
+
+	faction = "rabbit"
+	maxHealth = 30
+	health = 30
+
+	response_help = "pats"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits"
+
+	harm_intent_damage = 5
+	melee_damage_lower = 1
+	melee_damage_upper = 3
+	attacktext = list("nipped")
+	
+	movement_cooldown = 3
+
+	say_list_type = /datum/say_list/rabbit
+	ai_holder_type = /datum/ai_holder/simple_mob/passive
+	
+	// Vore vars
+	vore_active = 1
+	vore_bump_chance = 10
+	vore_bump_emote	= "playfully lunges at"
+	vore_pounce_chance = 40
+	vore_pounce_maxhealth = 100 // They won't pounce by default, as they're passive. This is just so the nom check succeeds. :u
+	vore_default_mode = DM_HOLD
+	vore_icons = SA_ICON_LIVING
+	
+	var/body_color 				//brown, black and white, leave blank for random
+	
+	var/grumpiness 				// This determines how grumpy we are. Pet us to increase it, leave us alone to decrease.
+	var/last_pet				// This tracks the last time someone patted us.
+	var/grump_decay = 5 SECONDS // This is how quickly our grumpiness decays.
+	
+/mob/living/simple_mob/vore/rabbit/New()
+	. = ..()
+	
+	if(!body_color)
+		body_color = pick( list("brown","black","white") )
+	icon_state = "rabbit_[body_color]"
+	item_state = "rabbit_[body_color]"
+	icon_living = "rabbit_[body_color]"
+	icon_dead = "rabbit_[body_color]_dead"
+	icon_rest = "rabbit_[body_color]_rest"
+	
+/mob/living/simple_mob/vore/rabbit/Life()
+	. = ..()
+	
+	if(grumpiness > 0 && last_pet > (world.time + grump_decay))
+		grumpiness = max(0, grumpiness-rand(5,10)) // Subtract grumpiness randomly in a range of 5-10 if we've not been PAT in the last 5 seconds.
+		
+/mob/living/simple_mob/vore/rabbit/examine(mob/user)
+	. = ..()
+	
+	switch(grumpiness)
+		if(-INFINITY to 25)
+			desc += "They appear calm."
+		if(26 to 50)
+			desc += "They seem slightly annoyed."
+		if(51 to 75)
+			desc += "They seem very annoyed, perhaps you should stop petting them."
+		if(75 to INFINITY)
+			desc += "They are very angry. Petting them will likely result in unpleasant things."
```

Comment:
These should probably be ". +=" so they don't end up with really long descriptions after being examined a lot.